debian/rules

```diff
@@ -67,7 +67,7 @@ clean:
 	  ltmain.sh missing
 	rm -f configure Makefile.in man/Makefile.in src/Makefile.in
 	rm -f src/acconfig.h.in
-	rm -f debian/ceph-base.ceph.init debian/radosgw.init debian/ceph.logrotate debian/radosgw.logrotate
+	rm -f debian/ceph-base.ceph.init debian/radosgw.init debian/ceph-common.logrotate
```

Comment:
no, these lines are how "make clean" is supposed to cleanup the package maintainer's building directory. it has nothing to do with the end user.